<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableWidget" name="leaderboard_tbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>421</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    QTableWidget {
        border: 1px solid lightgrey;
        border-radius: 4px;
        gridline-color: lightgrey;
        background-color: #f9f9f9;
    }
    QHeaderView::section {
        background-color: black;
        color: white;
        padding: 5px;
        font-size: 14px;
        border: none;
    }
    QTableWidget::item {
        padding: 5px;
    }
    QTableWidget::item:selected {
        background-color: #d9d9d9;
        color: black;
    }</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>134</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name:</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Score:</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NewTime:</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>41</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px;
	border: 1px solid black;
	font: 12pt &quot;Franklin Gothic Medium&quot;;
}

QPushButton:hover {
	border-radius: 20px;
	border: 2px solid black;
}

QPushButton:pressed {
	border-radius: 20px;
	border: 2px solid black;
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Heavy</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Leaderboard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>99</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
        border: 1px solid black;  /* Default black border */
        border-radius: 3px;  /* Rounded corners */
        padding: 5px;  /* Add some padding */
    }
QLineEdit:hover {
        border: 1px solid lightgrey;  /* Light grey border on hover */
    }
QLineEdit:focus {
        border: 1px solid #0078d7;  /* Blue border on focus */
    }

</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
